modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+When you run a command in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as a task to that namespace.
+
+.Procedure
```

Comment:
It seems to me that his procedure is to use pipeines to run the buildah container as a root user. Should the name of this doc reflect that?

Reply:
@HarshCasper, if Jake agrees with the above suggestion, you can mark this thread as resolved.